src/main/java/org/folio/service/file/cleanup/StorageCleanupServiceImpl.java

```diff
@@ -44,6 +46,7 @@ private Date getFileDefinitionExpirationDate() {
   }
 
   private Future<CompositeFuture> deleteExpiredFilesAndRelatedFileDefinitions(List<FileDefinition> fileDefinitions, String tenantId) {
+    LOGGER.info(format("Start process of removing files and file definitions, number of file definitions to clean up: %s", fileDefinitions.size()));
```

Comment:
Ideally we should be using the parameterized logging offered by log4j : https://logging.apache.org/log4j/2.x/performance.html#asyncLoggingWithParams

Reply:
@KVupp thanks, done